exchanges/gateio/gateio_websocket_test.go

```diff
@@ -0,0 +1,176 @@
+package gateio
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/thrasher-corp/gocryptotrader/currency"
+	exchange "github.com/thrasher-corp/gocryptotrader/exchanges"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/account"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/asset"
+)
+
+func TestProcessSpotBalances(t *testing.T) {
+	t.Parallel()
+	e := new(Exchange) //nolint:govet // Intentional shadow
+	e.SetDefaults()
+	e.Name = "ProcessSpotBalancesTest"
+
+	ctx := account.DeployCredentialsToContext(t.Context(), &account.Credentials{Key: "test", Secret: "test"})
+
+	// t.Parallel() not required below as this is sequenced. Some timestamps are identical from trading activity.
```

Comment:
This comment makes no sense 😝 
I think it means `t.Run`.
But I also think it needs to live inside the body of the `for` block, because that's where you'd put the `t.Run`
```suggestion
	// Sequential tests, do not use t.Run(); Some timestamps are deliberately identical from trading activity
```